modules/layers/src/polygon-layer/polygon-layer.ts

```diff
@@ -18,12 +18,186 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import {CompositeLayer, createIterable, log} from '@deck.gl/core';
+import {
+  Accessor,
+  AccessorFunction,
+  Color,
+  CompositeLayer,
+  createIterable,
+  Layer,
+  LayersList,
+  log,
+  Unit,
+  UpdateParameters
+} from '@deck.gl/core';
 import SolidPolygonLayer from '../solid-polygon-layer/solid-polygon-layer';
 import PathLayer from '../path-layer/path-layer';
 import * as Polygon from '../solid-polygon-layer/polygon';
 import {replaceInRange} from '../utils';
 
+// TODO: not sure where this type belongs to
```

Comment:
This should be defined by core's `lighting` shader module. You can leave it as is for now.